Marlin/example_configurations/delta/kossel_pro/Configuration_adv.h

```diff
@@ -0,0 +1,645 @@
+#ifndef CONFIGURATION_ADV_H
```

Comment:
@Wackerbarth

And lhere, too:

```
// Example configuration file for Vellemann K8200
// tested on K8200 with VM8201 (Display)
// and Arduino 1.6.1 (Win) by @CONSULitAS, 2015-04-14
// https://github.com/CONSULitAS/Marlin-K8200/archive/K8200_stable_2015-04-14.zip
```

Please compare to Marlin\example_configurations\K8200\.